express/views/index.jade

```diff
@@ -6,7 +6,7 @@ block content
       h1= title
       p Welcome to #{title}
       // Container for the react application
-      div#react-app
+      div#react-app(data-host="http://localhost:3000/api/v1")
```

Comment:
I would define it in a constants / config file, so we have a single place where the url can be configured.